src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/EventSource.cs

```diff
@@ -3883,6 +3886,15 @@ private bool SelfDescribingEvents
         internal const string s_ActivityStartSuffix = "Start";
         internal const string s_ActivityStopSuffix = "Stop";
 
+        internal const string MultipleEventSourcesSwitch = "System.Diagnostics.Tracing.EventSource.AllowMultipleEventSources";
```

Comment:
Naming nit: interpreting the name litterally this is already allowed. It is duplicate EventSource names that are at issue
```suggestion
        internal const string AllowDuplicateSourceNamesSwitch = "System.Diagnostics.Tracing.EventSource.AllowDuplicateSourceNames";
```